content/conditions/warts/main-content.md

```diff
@@ -1,56 +1,62 @@
-## Warts often go away on their own
+Warts don’t cause you any harm but some people find them itchy, painful or embarrassing.
+Verrucas are more likely to be painful - like standing on a needle.\n\nYou can treat warts if they’re bothering you,
```

Comment:
`\n\n\` need removing and either putting on a new line or replacing with a space.